app/src/main/java/ar/com/wolox/android/cookbook/twitterlogin/TwitterLoginRecipePresenter.kt

```diff
@@ -130,25 +153,7 @@ class TwitterLoginRecipePresenter @Inject constructor(
     }
 
     private fun defaultTwitterLogin() {
-        if (getTwitterSession() == null) {
-            view.setLoginCallback(twitterAdapter.twitterCallback(object : TwitterLoginAuthListener {
-                override fun onAuthSuccess(result: TwitterSession) {
-                    fetchTwitterEmail(result)
-                }
-
-                override fun onAuthError(message: String?) {
-                    if (message != null) {
-                        view.showError(message)
-                    } else {
-                        view.showAuthFail()
-                    }
-                }
-
-                override fun onAuthFail() {
-                    view.showAuthFail()
-                }
-            }))
-        } else {
+        if (getTwitterSession() != null) {
```

Comment:
We are using Kotlin, you can do:`getTwitterSession()?.let{fetchTwitterEmail(getTwitterSession())}` instead